### ISSUE TITLE
What is difference between docs:generate:ordered & docs:generate ?

### ISSUE THREAD

**Issue by pramodshinde** [zipmark/rspec_api_documentation#250]
Hello, 
Scenario 1: 
```ruby
config.keep_source_order = false
```
I tried 
```ruby
rake docs:generate
rake docs:generate:ordered 
```
For both above commands same documented is getting generated i.e ordered by description 

Scenario 2: 
```ruby
config.keep_source_order = true
```
I tried 
```ruby
rake docs:generate
rake docs:generate:ordered 
```
For both above commands same documented is getting generated i.e ordered by source 

In both scenario I can't find any difference between  `rake docs:generate:ordered` & `rake docs:generate`, Am I missing something here?

**Comment by anilmaurya** [comment]
@pramodshinde I read `Rake Task`(https://github.com/zipmark/rspec_api_documentation#rake-task) section of README and I can only see 

```
$ rake docs:generate
```

Can you point out where you read docs for `rake docs:generate:ordered` ?

**Comment by oestrich** [comment]
The `rake docs:generate:ordered` task is [here](https://github.com/zipmark/rspec_api_documentation/blob/master/lib/tasks/docs.rake#L10). I think the problem may have started when we switched to rspec 3, someone just did a PR fixing the ordered task, #245.

**Comment by anilmaurya** [comment]
Docs should be updated with available options.

**Comment by pramodshinde** [comment]
@anilmaurya  Yes ultimately I wanted to update docs to include both tasks, before that I tested both tasks I did not find any difference between above mentioned tasks, So wanted confirm is there any difference or its an issue?

**Comment by oestrich** [comment]
With the new version (4.7.0) does this still happen. It has PR #245 merged.
